tests/spirograph.c

```diff
@@ -9,6 +9,7 @@
 #include <stdint.h>
 #include <stdio.h>
 #include <stdlib.h>
+#include <unistd.h>
```

Comment:
Could you please clarify what issue this patch addresses, such as compile warnings or errors?

Reply:
GCC complains without including `unistd.h`

```shell
tests/spirograph.c:177:5: warning: implicit declaration of function ‘usleep’ [-Wimplicit-function-declaration]
```